test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -66,6 +66,12 @@ SMTCheckerTest::SMTCheckerTest(string const& _filename): SyntaxTest(_filename, E
 		m_ignoreCex = true;
 	else
 		BOOST_THROW_EXCEPTION(runtime_error("Invalid SMT counterexample choice."));
+
+	[[maybe_unused]] auto const& os = m_reader.stringSetting("SMTIgnoreOS", "none");
+#ifdef __APPLE__
+	if (os == "osx")
+		m_shouldRun = false;
+#endif
```

Comment:
So this PR also partially implements #11870? :)

Why not add Linux and Windows as an option and extract it into a separate PR?

Reply:
Yea could do that, I did it here mostly because it was quick and I wanted to see if it would make the tests pass.